<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgFilterFastColorOverlay</class>
 <widget class="QWidget" name="WdgFilterFastColorOverlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Fast Color Overlay</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Color:</string>
        </property>
        <property name="buddy">
         <cstring>cmbCompositeOp</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KisColorButton" name="bnColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Blend Mode:</string>
        </property>
        <property name="buddy">
         <cstring>cmbCompositeOp</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KisCompositeOpComboBox" name="cmbCompositeOp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Set the blend mode for the layer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Opacity:</string>
        </property>
        <property name="buddy">
         <cstring>intOpacity</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KisSliderSpinBox" name="intOpacity" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>15</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set the master opacity for the overlay color</string>
        </property>
        <property name="whatsThis">
         <string>Adjust the transparency of the overlay color</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>151</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisColorButton</class>
   <extends>QPushButton</extends>
   <header>kis_color_button.h</header>
  </customwidget>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QWidget</extends>
   <header location="global">kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisCompositeOpComboBox</class>
   <extends>QComboBox</extends>
   <header>kis_cmb_composite.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
